src/Microsoft.AspNetCore.Mvc.RazorPages/Infrastructure/PageActionDescriptorProvider.cs

```diff
@@ -109,5 +113,42 @@ private void AddActionDescriptors(IList<ActionDescriptor> actions, PageRouteMode
                 actions.Add(descriptor);
             }
         }
+
+        private static string TransformPageRoute(PageRouteModel model, string pageRoute)
+        {
+            model.Properties.TryGetValue(typeof(IOutboundParameterTransformer), out var transformer);
+            var pageRouteTransformer = transformer as IOutboundParameterTransformer;
+
+            if (pageRouteTransformer == null)
```

Comment:
You can `is` this for #PranavPoints

Reply:
`is null`?

***!!! SQL flashbacks intensifies !!!***

Reply:
`if (obj is IOutboound.... transformer)`...

FYI Mads is telling folks to use `is null` now, I have really mixed feelings about it.

Reply:
You still have not claimed your PranavPoints!! /cc @pranavkm

Reply:
This felt weird:

```cs
if (!(transformer is IOutboundParameterTransformer pageRouteTransformer))
{
    return xxx;
}
```